docs/src/main/sphinx/admin/properties-http-client.rst

```diff
@@ -0,0 +1,312 @@
+======================
+HTTP client properties
+======================
+
+``http-client.https.hostname-verification``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:** ``true``
+
+Verifies that the server hostname matches the server certificate.
+
+``http-client.http2.enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:**
+
+.. No default value?
+
+Enables the HTTP/2 transport.
+
+``http-client.connect-timeout``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Minimum value:** ``0ms``
+* **Default value:** ``5s``
+
+``http-client.request-timeout``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Minimum value:** ``0ms``
+* **Default value:** ``5m``
+
+``http-client.idle-timeout``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Minimum value:** ``0ms``
+* **Default value:** ``1m``
+
+``http-client.max-connections``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Minimum value:** ``1M``
+* **Default value:** ``200``
+
+``http-client.max-connections-per-server``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Minimum value:** ``1M``
+* **Default value:** ``20``
+
+``http-client.max-requests-queued-per-destination``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Minimum value:** ``1M``
```

Comment:
So this is similarly one where a minimum doesn't make a ton of sense (there's potentially an argument for 0? but that'd be very not-fault-tolerant).

But this is the queue, so you've already got max connections defined, and once the max number of connections has been reached, pending connections go into a queue. This configures how big that queue can get before it just abandons everything and throws an error.